js/views/itemEditVw.js

```diff
@@ -663,5 +663,10 @@ module.exports = baseVw.extend({
     this.validateDescription();
 
     return this.$('#contractForm')[0].checkValidity();
+  },
```

Comment:
The clone only fires if it's editing an existing item. When it's a new item the close doesn't happen, but the data for image_hashes gets into the new model that is created.

Somehow the defaultItem object is getting set to the values being set in editItem, or at least the image_hash is. 

Maybe there's a reference somewhere that should be a copy.